.travis.yml

```diff
@@ -77,7 +77,7 @@ install:
   - echo "CXX=${CXX}"
 
 before_script:
-  - ./autogen.sh
+  - ./autogen.sh --enable-debug
```

Comment:
Not convinced that it implies a broader test coverage: few extra checks are done in DEBUG mode (mostly division by zero and dimension checks), on the other hand, I rember hitting several times bugs that only showed up with -O2 optimization flag, for instance the dekker mutliplication, using x87 80bits multiplier in -O0 and SSE 64bits multiplier in -O2.

Reply:
Reverted in e3c38a1